content/docs/lists-and-keys.md

```diff
@@ -130,17 +130,17 @@ const todoItems = todos.map((todo, index) =>
 );
 ```
 
-We don't recommend using indexes for keys if the order of items may change. This can negatively impact performance and may cause issues with component state. Check out Robin Pokorny's article for an [in-depth explanation on the negative impacts of using an index as a key](https://medium.com/@robinpokorny/index-as-a-key-is-an-anti-pattern-e0349aece318). If you choose not to assign an explicit key to list items then React will default to using indexes as keys.
+यदि आइटम का क्रम बदल सकता है तो हम key के लिए इंडेक्स का उपयोग ना करने की सिफारिश करते हैं। इसका नकारात्मक प्रभाव पड़ सकता हैं और यह कौम्पोनॅन्ट की अवस्था के साथ समस्या भी पैदा कर सकता है। अधिक जानकारी के लिए Robin Pokorny के [इंडेक्स को key के रूप में उपयोग करने के नकारात्मक प्रभाव](https://medium.com/@robinpokorny/index-as-a-key-is-an-anti-pattern-e0349aece318) के आर्टिकल को देखें। यदि आप लिस्ट आइटम्स को एक explicit key नहीं देते हैं, तो React key के रूप में इंडेक्स का उपयोग करता है।
 
-Here is an [in-depth explanation about why keys are necessary](/docs/reconciliation.html#recursing-on-children) if you're interested in learning more.
+यदि आप अधिक सीखने में रुचि रखते हैं, तो यहाँ [key की आवशयकताओ के बारे में गहराई से स्पष्टीकरणं](/docs/reconciliation.html#recursing-on-children), दिया गया हैं।
 
-### Extracting Components with Keys {#extracting-components-with-keys}
+### Key के साथ कौम्पोनॅन्टस निकालना {#extracting-components-with-keys}
 
-Keys only make sense in the context of the surrounding array.
+Keys केवल उनके array के लिए ही होती है।
```

Comment:
incorrect translation

Reply:
I am not getting it's correct translation . Please suggest it's hindi translation.

Reply:
Keys केवल आस-पास के array के संदर्भ में ही समझ में आती हैं।